src/util/tev_curl.c

```diff
@@ -35,7 +35,8 @@
 #include "util/tev_curl.h"
 
 #define TCURL_IOBUF_CHUNK   1024
-#define TCURL_IOBUF_MAX    16384
+/* This limit in the same one as KCM_REPLY_MAX */
+#define TCURL_IOBUF_MAX     10*1024*1024
```

Comment:
So I'm fine with this change, but please do not close the related ticket. The reason is that the secrets "back end" of KCM can in theory transfer much bigger payloads, for example when listing all secrets. I think we should dynamically adjust the limit and the limit should be passed to the tcurl module from the caller.